packages/discord-player/src/utils/QueryResolver.ts

```diff
@@ -211,18 +211,11 @@ class QueryResolver {
       if (DomainsMap.YouTube.includes(url.host)) {
         query = query.replace(/(m(usic)?|gaming)\./, '').trim();
         const playlistId = url.searchParams.get('list');
-        const videoId = url.searchParams.get('v');
-        if (playlistId) {
-          if (videoId && playlistId.startsWith('RD'))
```

Comment:
this was explicitly done for remixes, we dont want the extractors taking playlists as a video in other cases.

Reply:
oh my bad, you want to begin from the `v`'s index. Since discord-player does not handle this, we should assume extractors are doing it correctly. And at the moment, only @retrouser955's extractor is known to be youtube compatible, I guess we can add this.